units/INU2003/INU2003_unit.bp

```diff
@@ -62,7 +61,6 @@ UnitBlueprint {
         'LAND',
         'TECH2',
         'INDIRECTFIRE',
-        'SILO',

```

Comment:
then just add it back by pushing a commit into this branch which adds the line back